lib/_stream_writable.js

```diff
@@ -190,7 +190,7 @@ if (typeof Symbol === 'function' && Symbol.hasInstance) {
     }
   });
 } else {
-  realHasInstance = function(object) {
+  realHasInstance = function realHasInstance(object) {
```

Comment:
This i not needed, the name is inferred from the variable name.

Reply:
Thanks, removed